### PR TITLE
new LOs gets added to all students in that cohort

### PR DESCRIPTION
**Pull request by zaynshah**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -194,3 +189,11 @@ async function makeSession(userID, e, server, isAdmin) {
...
   server.setCookie({ name: "isAdmin", value: isAdmin, expiryDate });
 }
+
+const check = [
+  ...db.query(
+    "SELECT DISTINCT(users.email), users.cohort_id, users.id FROM learning_objectives JOIN users ON users.cohort_id = learning_objectives.cohort_id WHERE users.cohort_id=1"
+  ),
+];
+
+console.log(check);
```
Comment (olliecase-green) [inline server.js]: Is the console log necessary?

**Review by olliecase-green — APPROVED** [review]
Looks good

File: server.js [hunk 2]
```diff
@@ -73,7 +67,7 @@ async function getStudentData(server) {
   SELECT *
   FROM results
   WHERE user_id = ?
-  ORDER BY id ASC
+  ORDER BY topic ASC
```
Comment (FahmidulHaque97) [inline server.js:70]: Good changes to the visual rendering on Admin side.

File: server.js [hunk 3]
```diff
@@ -110,6 +104,19 @@ async function postLO(server) {
...
+
+  check.forEach((i) =>
+    db.query(`INSERT INTO results (user_id,email,cohort_id,topic,learning_objective) VALUES ('${i[2]}','${i[0]}','${i[1]}',?,?)`, [
+      topic,
+      learning_objective,
+    ])
+  );
 }
 
 function validateEmail(email) {
```
Comment (FahmidulHaque97) [inline server.js:122]: Maybe move this function to the bottom to make codebase cleaner.

**Review by FahmidulHaque97 — COMMENTED** [review]
Great work on adding new Learning Objectives to all students within one cohort.

**Pull request merged**